cross-chain/polygon/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "target": "ES2018",
+    "module": "commonJS",
+    "downlevelIteration": true,
+    "esModuleInterop": true,
+    "resolveJsonModule": true
+  },
+  "files": ["./hardhat.config.ts"],
+  "include": ["./parentchain", "deploy_sidechain", "./test", "./typechain"]
```

Comment:
Shouldn't we have `./deploy_parentchain` instead of `./parentchain`?

Also, shouldn't we have `./deploy_sidechain` instead of `deploy_sidechain`?

Reply:
[03837dc](https://github.com/keep-network/tbtc-v2/pull/588/commits/03837dc2418c4027edc192727a6d2f025bc4d5e9)